JS/Lesson30/savas/my-app/src/components/Ex6/index.jsx

```diff
@@ -0,0 +1,14 @@
+// ## Exercise 6: Prevent Default
+// Design a link inside a component. Implement an event handler that prevents the default action when the link is clicked.
+
+export const Ex6 = () => {
+
+    function handleClick (event) {
+        event.preventDefault();
+        console.log("Button Clicked!")
+    }
+
+    return (
+        <button onClick={handleClick}>Prevent Default</button>
```

Comment:
in web development, when we say 'link' element we mean <a> element. Clicking on the button would not have the same effect as clicking on the link. The default behaviour of the link click would be redirection so we want to prevent it here.

Reply:
Ouch, I misunderstand it. I changed the button element with a link element.